ui/components/app/collectibles-items/collectibles-items.js

```diff
@@ -24,114 +24,149 @@ const width =
   getEnvironmentType() === ENVIRONMENT_TYPE_POPUP
     ? BLOCK_SIZES.ONE_THIRD
     : BLOCK_SIZES.ONE_SIXTH;
-export default function CollectiblesItems({ collections = {} }) {
-  const defaultDropdownState = {};
+
+const PREVIOUSLY_OWNED_KEY = 'previouslyOwned';
```

Comment:
[nit] Add an extra line here so it's easier to see where the default export of this file starts?

Reply:
done here https://github.com/MetaMask/metamask-extension/pull/13110/commits/f18e1eea775cc11d5395071ec8294f2206b93fac